core/src/test/scala/cromwell/core/Tags.scala

```diff
@@ -6,4 +6,5 @@ object Tags {
   object DockerTest extends Tag("DockerTest")
   object IntegrationTest extends Tag("CromwellIntegrationTest")
   object DbmsTest extends Tag("DbmsTest")
+  object PostWomTest extends Tag("PostWomTest")
```

Comment:
ToL: Are we marking these with a tag so we can ignore them all at once? In which case, do we need to ignore them all individually?

Reply:
yeah I realized halfway through that we could ignore all the ones tagged with this but I was already halfway through so..